src/pg/brc20/brc20-pg-store.ts

```diff
@@ -160,9 +167,13 @@ export class Brc20PgStore extends BasePgStoreModule {
                   total_balance = brc20_total_balances.total_balance + EXCLUDED.total_balance
               )
             `
-      }
+      }, deploy_update AS (
+        UPDATE brc20_deploys
+        SET tx_count = tx_count + 1
+        WHERE id = (SELECT brc20_deploy_id FROM validated_transfer)
+      )
       INSERT INTO brc20_events (operation, inscription_id, genesis_location_id, brc20_deploy_id, transfer_id) (
-        SELECT 'transfer_send', id, ${location.id}, brc20_deploy_id, id
```

Comment:
@rafaelcr  Had a bug here that will have caused incorrect DB values. Should I add a migration to mitigate (rewrite the inscription_id column of `transfer_send`s?)